scm/driver/gitlab/testdata/webhooks/pull_request_reopen.json

```diff
@@ -37,7 +37,7 @@
     "merge_commit_sha": null,
     "merge_error": null,
     "merge_params": {
-      "force_remove_source_branch": false
+      "force_remove_source_branch": "0"
```

Comment:
Looking at the link, you don't need to change this, right? As it can be bool or string. May be better to add a test case where it's "0", and one where it's false and see that the code works...

Reply:
I changed some to "0" the rest is still false. So there are both in the Tests

Reply:
Cool, lgtm